functions/tests/utils/test_audio.py

```diff
@@ -19,12 +18,13 @@ def test_cut(tmp_path: Path):
     cut(
         file=audio,
         destination=cut_audio,
-        sample_rate=sample_rate,
         start_ms=start_ms,
         stop_ms=stop_ms,
     )
-    data, _ = sf.read(cut_audio)
-    assert len(data) == (stop_ms - start_ms) * sample_rate / 1000
+    with open(cut_audio, "rb") as f:
+        obj = wave.open(f, "rb")
```

Comment:
```suggestion
    with open(cut_audio, "rb") as audio_file:
        obj = wave.open(audio_file, "rb")
```

Reply:
I assume this works, but it does seem a little strange to have to open and read it in binary twice, could wave.open accept a file path, possibly?

Reply:
It does work from the tests, but it is very odd - the reasoning was so I could use the `with` and make sure that the files were definitely closed, but I think this might be lazy superstition. I'll remove the outer `with`s in the next commit.

Reply:
FIxed in latest revision.